diff --git a/server/query-routes.js b/server/query-routes.js
index 49feae7..3f83c1c 100644
--- a/server/query-routes.js
+++ b/server/query-routes.js
@@ -6,7 +6,7 @@ var queryCache = require("./query-cache");
 var auth = require("./auth");

 var connect = function(cb) {
-  mongo.connect("mongodb://localhost/seismo", { useNewUrlParser: true }, cb);
+  mongo.connect("mongodb://seismo-database/seismo", { useNewUrlParser: true }, cb);
 };

 router.get("/stations", function(req, res, next) {
